langs/berry/Dockerfile

```diff
@@ -1,18 +1,20 @@
-FROM alpine:3.21 AS builder
+FROM alpine:3.22 AS builder
 
-RUN apk add --no-cache curl gcc make musl-dev python3 readline-dev
+RUN apk add --no-cache build-base curl python3 readline-dev
 
-RUN curl -L https://github.com/berry-lang/berry/archive/97e0fca.tar.gz \
-  | tar xz && mv berry-* berry
+ENV VER=1.1.0
```

Comment:
I'm not sure we can downgrade without breaking the language, see 1302ee506ba05afd8f096ad4d633ec33f6615a26